src/main/java/org/sonatype/cs/metrics/service/LoaderService.java

```diff
@@ -93,7 +93,8 @@
 	public boolean loadMetricsFile(String fileName, String header, String stmt) throws IOException {
 		boolean status = false;
 		
-		String filePath = dataDir + "/" + fileName;
+		String filePath = Paths.get(System.getProperty("user.dir")).resolve(Paths.get(dataDir).resolve(fileName)).toString();
```

Comment:
Perhaps a bit beyond the scope of initial tests. We should add an epic for specific code-cleanup/technical debt tasks like using dynamic or configured variables. Richard and I spoke about this, leaving this comment for higher visibility for everyone else.

Reply:
Agreed